src/KubernetesClient/Kubernetes.ConfigInit.cs

```diff
@@ -58,7 +58,9 @@ public Kubernetes(KubernetesClientConfiguration config, params DelegatingHandler
                         throw new KubeConfigException("a CA must be set when SkipTlsVerify === false");
                     }
 
-                    using (System.IO.MemoryStream certStream = new System.IO.MemoryStream(config.SslCaCert.RawData))
+                    var caCert = CaCert[CaCert.Count - 1];
```

Comment:
could it support collection here?

Reply:
I'm not familiar with the Java libraries, but I changed it to at least add all the certs in the bundle to the trusted certificate list.